### PR TITLE
Add new status changes and update related items

### PR DESCRIPTION
**Pull request by Kenpachi2k13**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Add 7 new status changes and use them for the related items.

Closes #1177

**Issues addressed:** None.

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: db/pre-re/item_db.conf [hunk 1]
```diff
@@ -74203,6 +74303,33 @@ item_db: (
...
+		notrade: true
+		noselltonpc: true
+		nocart: true
+		nogstorage: true
+		nomail: true
+		noauction: true
+	}
+	Script: <"
+		specialeffect(EF_BASH3D, AREA, playerattached());
+		sc_start(SC_FOOD_STR, 1800000, 10);
```
Comment (Asheraf) [inline db/pre-re/item_db.conf]: This should use the `SC_FOOD_*_CASH` variant

File: db/sc_config.conf [hunk 2]
```diff
@@ -3675,3 +3675,24 @@ SC_MADOGEAR: {
...
+SC_SKF_ATK: {
+	Icon: "SI_SKF_ATK"
+}
+SC_SKF_ASPD: {
+	Icon: "SI_SKF_ASPD"
+}
+SC_SKF_CAST: {
+	Icon: "SI_SKF_CAST"
+}
+SC_ALMIGHTY: {
```
Comment (Asheraf) [inline db/sc_config.conf]: `NoDeathReset, NoDispelReset, NoClearanceReset` Flags are required and please check the flags for other status in the aegis [EFSTconfig.lua](https://pastebin.com/LuyhJNUv)

File: db/pre-re/item_db.conf (hunk 1 above)
Comment (Kenpachi2k13) [reply db/pre-re/item_db.conf]: Done.

File: db/sc_config.conf (hunk 2 above)
Comment (Kenpachi2k13) [reply db/sc_config.conf]: Done.

**Pull request merged**